Package.swift

```diff
@@ -0,0 +1,25 @@
+// swift-tools-version:5.0
```

Comment:
Probably only this comment is needed 🧹

Reply:
Nice! I just did the default project init.

Reply:
Doesn't work if I only leave one line in this file 😞 

<img width="598" alt="Screenshot 2020-04-22 at 13 10 04" src="https://user-images.githubusercontent.com/7569279/79969761-9c47c280-849a-11ea-8fce-3a8945bf16fb.png">

Reply:
One line as in only that comment with rest of package description :D